<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>protectormain</class>
 <widget class="QDialog" name="protectormain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>兰老师保护系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pic.qrc">
    <normaloff>:/new/prefix1/001.ico</normaloff>:/new/prefix1/001.ico</iconset>
  </property>
  <widget class="QPushButton" name="Enter">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>进入系统</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AdminMenu">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>管理员菜单</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>兰老师保护系统</string>
   </property>
  </widget>
  <widget class="QLabel" name="Status">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>当前处于保护模式</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>倒计时</string>
   </property>
  </widget>
  <widget class="QLabel" name="CountDown">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>21</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ToProtectMode">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>保护模式</string>
   </property>
  </widget>
  <widget class="QPushButton" name="KeepData">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>保留数据</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="pic.qrc"/>
 </resources>
 <connections/>
</ui>
